src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import core.basesyntax.generator.FigureSupplier;
+import java.util.Random;
+
+public class Main {
+    public static void main(String[] args) {
+        Random random = new Random();
+        int figureCount = random.nextInt(); // You can change it if you want
```

Comment:
please remove commented code and set some upper limit for nextInt()
remember that magic numbers should be constants